fling/dataset/domainnet.py

```diff
@@ -0,0 +1,93 @@
+import torch.utils.data as data
+import torchvision.transforms as transforms
+import os
+from PIL import Image
+import numpy as np
+
+from fling.utils.registry_utils import DATASET_REGISTRY
+
+
+@DATASET_REGISTRY.register('domainnet')
+class DomainNetDataset(data.Dataset):
+    r"""
+    Implementation for DomainNet dataset. 
+    """
+    def __init__(self, cfg: dict, domain: str,  train: bool): # cfg can be loaded
+        super(DomainNetDataset, self).__init__()
+        self.train_transform = transforms.Compose([
+        transforms.Resize([256, 256]),
+        transforms.RandomHorizontalFlip(),
+        transforms.RandomRotation((-30, 30)),
+        transforms.ToTensor()
+    ])
+        self.test_transform = transforms.Compose([
+        transforms.Resize([256, 256]),
+        transforms.ToTensor()
+    ])
+        self.train = train
+        self.cfg = cfg
+        self.train_imgs, self.train_labels, self.test_imgs, self.test_labels = \
```

Comment:
Load train_imgs and labels when ``train==True`` and otherwise load test_imgs and labels. Do not load both of them simultaneously